### PR TITLE
Adding NodeJS version 12 env

### PR DESCRIPTION
**Pull request by vishal-biyani**
This change is [Reviewable](https://reviewable.io/reviews/fission/fission/1683)

### CODE REVIEW

**Comment by vishal-biyani** [comment]
Tested the XML example for this: https://github.com/fission/fission/pull/1541 and with this environment, it works great. I was able to get an entire POM.xml tested against Java env without any change: 
image

File: environments/nodejs/Dockerfile-12.16 [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# A docker image for the func container.
+
+# default variant is the official alpine node image (much smaller than the standard image)
```
Comment (vishal-biyani) [inline environments/nodejs/Dockerfile-12.16:3]: Not super relevant comments, let's remove these.

File: environments/nodejs/builder/Dockerfile-12.16 [hunk 2]
```diff
@@ -0,0 +1,11 @@
+ARG BUILDER_IMAGE=fission/builder
+FROM ${BUILDER_IMAGE}
+# default variant is the official alpine node image (much smaller than the standard image)
+FROM node:12.16-alpine3.11
```
Comment (vishal-biyani) [inline environments/nodejs/builder/Dockerfile-12.16:4]: The latest LTS on website seems to be 12.18.3 I am not sure how big the difference is, can you please confirm if we are ok with 12.16 or should bump to higher versions?

**Comment by vishal-biyani** [comment]
Hey @therahulbhati - looks good except some minor comments!

File: environments/nodejs/Dockerfile-12.16 (hunk 1 above)
Comment (therahulbhati) [reply environments/nodejs/Dockerfile-12.16]: Done

**Comment by vishal-biyani** [comment]
Thanks for getting this done @therahulbhati - LGTM!

**Pull request merged**